src/components/language/language.resolver.ts

```diff
@@ -54,7 +54,7 @@ export class LanguageResolver {
     description: `The fiscal year of the sponsor date`,
```

Comment:
```suggestion
    description: `The fiscal year of the sponsor start date`,
```

Reply:
Actually can you just remove this field @zkhin? I've changed my mind that value can just be calculated by the consumer.

Reply:
So you mean I have to remove the origin `sponsorDate` field?

Reply:
No I mean remove the `beginFiscalYear` resolver

Reply:
I see.